### ISSUE TITLE
Not able to use both TOTP and OpenID authntication

### ISSUE THREAD

**Issue by sk8er000** [abesnier/docker-guacamole#44]

**Comment by abesnier** [comment]
Sorry for the delay in the reply, I was busy with my full time job.

This is marked as fixed with version 1.6.0, so that's what the `github` images are made for : Guacamole client and server components are built from the available code on github, and changes for 1.6.0 are usually merged very quickly. This specific pull change has been merged into master quite a long time ago, so that should work.

So you should try one of the `github` images, matching your postgresql version

**Comment by sk8er000** [comment]
No problem, I've deployed a new container using abesnier/guacamole:latest-pg16 and I've migrated my database.
The guacamole version is still 1.5.5 and the bug is still there, am I missing something?

Thank you for your help

**Comment by abesnier** [comment]
Yes, because the fix is available for guacamole 1.6.0, and only the `github` images use this version.

`latest` uses the files made available by Apache ([details here](https://guacamole.apache.org/releases/1.5.5/)), and this is still version 1.5.5.

The Guacamole team is finalizing 1.6.0 (still a few issues to be fixed and documentation to be updated).

If you need this fixed asap, you should try one of the `github` images (in your case `github-pg16` so you don't have to backup and restore your configuration database).

Cheers
Antoine
